Converts Portfolio.jsx to Function Component & Renders Content from an Array

infodsagar opened the pull request:
##Converts Portfolio.jsx to Function Component & Renders content from an array

-To make the code cleaner and organized Component import was removed

-Removes Component import no longer needs to do class extension

-Class components converted to function keeps code clean and easy to understand

-Stores Portfolio function in const & removes render method

-Creates ProjectData.jsx to store project data in an Array

-Renders over projectData using  map() method to loop over an Array, and reduce repetitions of the same code.

-Special thanks to Botan for preparing an Array file.

- Fixes #96 

- Fixes #31

infodsagar committed:
Creates ProjectData.jsx

infodsagar committed:
Creates ProjectData.jsx & Converts Portfolio.jsx to Function component

infodsagar committed:
Creates ProjectData.jsx & Converts Portfolio.jsx to Function component

foadbotan commented:
Hi Sagar, this looks great. It's very concise and readable. 

I'm happy to merge this, but there is one minor bug. The team array needs to be joined `team.join(", ")`.

infodsagar committed:
Fixes missing commas between team member list

davidtaing committed:
Rename ProjectData.jsx to ProjectData.js

Renames file to ProjectData.js to improve clarity as this file only contains a JS array, and does not have any JSX.

davidtaing committed:
Fix: Broken Import from Previous Change

Fixes the broken import from the rename from ProjectData.jsx to ProjectData.js.

davidtaing reviewed (approved):
Thanks for the review @foadbotan!

And thank your for your effort @infodsagar. Approved!

P.S. I've made a small change to rename the ProjectData.jsx file to ProjectData.js (because there's no React Components/JSX in there).

davidtaing commented:
@infodsagar, I forgot to mention that for your PR comments, it's okay not to tell us everything.

Try to tell us the big picture with a few key points and leave out the little things.
